doc/Img-guide/_read-me.txt

```diff
@@ -0,0 +1,2 @@
+This path is for recipient the illustrations of the guide.
+Which does not require translation for all
```

Comment:
I suggest that you call this file `README.md`. Then, it will get automatically picked up for rendering by GitHub when browsing the code. I would like to propose a better wording: "This directory shall hold the figures of the guide, which are shared among all translations."